### PR TITLE
[wip] fix clearlinux

### PR DESCRIPTION
**Pull request by RDIL**
## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **Adds a new image or tag**
  - [ ] Not Applicable
- **OR**
  - [x] Update [README.powershellcommunity.md](https://github.com/PowerShell/PowerShell-Docker/blob/master/assets/README.powershellcommunity.md)
  - [x] Update [vsts-ci.yml](https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml)

### CODE REVIEW

**Comment by TravisEz13** [comment]
@kiazhi Can you review?

**Comment by RDIL** [comment]
Haven’t really done stuff yet

**Comment by TravisEz13** [comment]
We don't have the WIP bot here... you can open draft PRs and it won't notify us to review until you ask for a review.

File: release/community-stable/clearlinux/docker/Dockerfile [hunk 1]
```diff
@@ -95,13 +95,10 @@ RUN \
...
           }" \
     # upgrade distro
-    && swupd update \
-    # clean downloaded packages
-    && swupd clean --all
+    && swupd update
 
 # Copy only the files we need from the previous stage after distro upgrade
 # These files are required to resolve SSL issue with Invoke-WebRequest
-COPY --from=installer-env ["/usr/lib64/libcrypto.so.1.0.0", "/usr/lib64/libcrypto.so.1.0.0"]
```
Comment (kiazhi) [inline release/community-stable/clearlinux/docker/Dockerfile]: @RDIL The libcrypto.so.1.0.0 is needed

Comment (RDIL) [reply release/community-stable/clearlinux/docker/Dockerfile]: The CI can't find it in the location specified

**Pull request closed without merging**